src/Analysis.php

```diff
@@ -219,6 +219,23 @@ public function getTraitsOfClass($class)
         return $definitions;
     }
 
+    public function getUsedTraits($class)
+    {
+        if ($class == '\\' || !class_exists($class)) {
+            return [];
+        }
+        $usedTraits = class_uses($class);
```

Comment:
This only works when the class/trait can be autoloaded.
When swagger-php cli is installed globally it doesn't execute any php code in the target project. Therefor we can reliably use reflection or class_exits etc.